promql/parser/printer_test.go

```diff
@@ -222,6 +252,8 @@ func TestExprString(t *testing.T) {
 		},
 	}
 
+	EnableExtendedRangeSelectors = true
```

Comment:
Do we want to disable that with `t.Cleanup()`?

Reply:
Yep why not :) Added that.